### DISCUSSION TITLE
Button "Send"

### DISCUSSION THREAD

**Discussion by f4emm** [foldynl/QLog#109] [General]
Hello,
Does anyone know what the "Send" button is for?
Thanks
Have a good day

**Comment by foldynl** [comment]
Hi,

did you mean DX Cluster's Send button? please, could you attach a screenshot?

Regards
Ladislav

**Reply by f4emm** [reply]
Yes it'is
Many thanks

**Reply by foldynl** [reply]
This button  sends the DXC command written in the DXC command line. This command can also be sent by pressing enter. It is a duplication that will be removed in the next version, because the currently prepared CW Console will also no longer have a "SEND" button. You can look forward to the next version.

**Reply by f4emm** [reply]
Hello,
Ok, I thought this button could possibly be used to spot a qso on the chosen dxcluster.
Thanks a lot
Have a good day

**Reply by foldynl** [reply]
Yeah.It is good idea to add "Send Spot" possibility. I will add it in the next version
